setup.py

```diff
@@ -16,6 +16,16 @@
 
 import kalite
 
+# Since pip 7.0.1, bdist_wheel has started to be called automatically when
```

Comment:
Is `pip` 7 a required upgrade to build a source dist, or is it backwards compatible with 6.11?